core/src/main/scala/spinal/core/Area.scala

```diff
@@ -69,59 +60,48 @@ trait Area extends NameableByComponent with ContextUser with OwnableRef with Sca
           component.setPartialName(name, childNamePriority)
           OwnableRef.proposal(component, this)
         }
-      case namable: Nameable =>
+      case namable: Nameable    =>
         if (!namable.isInstanceOf[ContextUser]) {
           namable.setPartialName(name, childNamePriority)
           OwnableRef.proposal(namable, this)
-        } else if (namable.asInstanceOf[ContextUser].component == component){
+        } else if (namable.asInstanceOf[ContextUser].component == component) {
           namable.setPartialName(name, childNamePriority)
           OwnableRef.proposal(namable, this)
         } else {
-          if(component != null) for (kind <- component.children) {
-            //Allow to name a component by his io reference into the parent component
+          if (component != null) for (kind <- component.children) {
+            // Allow to name a component by his io reference into the parent component
             if (kind.reflectIo == namable) {
               kind.setPartialName(name, childNamePriority)
               OwnableRef.proposal(kind, this)
             }
           }
         }
-      case _ =>
+      case _                    =>
```

Comment:
This is exactly what I was worried about

Reply:
Ahhh, machines are still decades away from human sensibilities XD
Let's drop automatic alignement